utils/io/fileutils/files.go

```diff
@@ -468,7 +468,9 @@ func CopyFile(dst, src string) (err error) {
 	if err != nil {
 		return err
 	}
-	return nil
+	// Preserve permissions of origin file
+	srcStat, _ := srcFile.Stat()
```

Comment:
The error returned from `srcFile.Stat()` should be returned with CheckError if not nil.